src/cai_lab_to_nwb/zaki_2024/interfaces/miniscope_imaging_interface.py

```diff
@@ -17,6 +18,101 @@
 from neuroconv.utils import DeepDict, dict_deep_update
 
 
+def get_session_start_time(folder_path: Union[str, Path]):
+    """
+    Retrieve the session start time from metadata in the specified folder.
+
```

Comment:
I also advice to name this `get_{something}_start_time` now that is not a method of the class or modify the docstring to make this clear.

Reply:
changed_ in get_recrtoding_start_time